plugins/draw-tools.js

```diff
@@ -681,11 +711,98 @@ window.plugin.drawTools.boot = function() {
 
   $('head').append('<style>' +
         '.drawtoolsSetbox > a { display:block; color:#ffce00; border:1px solid #ffce00; padding:3px 0; margin:10px auto; width:80%; text-align:center; background:rgba(8,48,78,.9); }'+
-        '.ui-dialog-drawtoolsSet-copy textarea { width:96%; height:250px; resize:vertical; }'+
+        '.ui-dialog-drawtoolsSet-copy textarea { width:96%; height:150px; resize:vertical; }'+
         '</style>');
 
 }
 
+// ---------------------------------------------------------------------------------
+// EXPORT (POLYS AS LINES)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.getDrawAsLines = function(){
+  var rawDraw = JSON.parse(window.localStorage[window.plugin.drawTools.KEY_STORAGE]);
+  var draw = [];
+
+  for(i in rawDraw){
+    var elemDraw = rawDraw[i];
+
+    if(elemDraw.type === 'polygon'){
+      var convElemDraw = {};
+      convElemDraw.color = elemDraw.color;
+      convElemDraw.type = 'polyline';
+      convElemDraw.latLngs = [];
+
+      var v = elemDraw.latLngs.length;
+      for(j in elemDraw.latLngs){
+        var ll = elemDraw.latLngs[j];
+        convElemDraw.latLngs.push(ll);
+      }
+      convElemDraw.latLngs.push(elemDraw.latLngs[0]);
+
+      draw.push(convElemDraw);
+    }else{
+      draw.push(elemDraw);
+    }
+  }
+
+  return JSON.stringify(draw);
+}
+
+// ---------------------------------------------------------------------------------
+// EMPTY POLYGONS (EMPTY DRAWN FIELDS)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.edf = {};
+window.plugin.drawTools.edf.storage = {};
```

Comment:
unused

Reply:
Sub-NameSpcae for EDF removed and intgrated with DT-Namespace